tools/test-clear.ts

```diff
@@ -0,0 +1,23 @@
+import os from 'node:os';
+import path from 'node:path';
+
+import chalk from 'chalk';
+import glob from 'fast-glob';
+import fs from 'fs-extra';
+
+(async () => {
+  // https://github.com/electron/forge/blob/v7.2.0/packages/utils/test-utils/src/index.ts#L24
+  const dirs = await glob(path.resolve(os.tmpdir(), 'electron-forge-test-*'));
```

Comment:
macOS CI seems to be failing due to this specific line of code now. If I revert to the previous state (using `os.homedir()`, the tests pass.

The problem seems to be fixed when I set `resolve.preserveSymlinks = true` in the renderer config, but I don't know if this is what we want in production.

```diff
diff --git a/packages/plugin/vite/src/VitePlugin.ts b/packages/plugin/vite/src/VitePlugin.ts
index 6e71ca7c6..e658bda55 100644
--- a/packages/plugin/vite/src/VitePlugin.ts
+++ b/packages/plugin/vite/src/VitePlugin.ts
@@ -193,6 +193,9 @@ the generated files). Instead, it is ${JSON.stringify(pj.main)}`);
   buildRenderer = async (): Promise<void> => {
     for (const userConfig of await this.configGenerator.getRendererConfig()) {
       await vite.build({
+        resolve: {
+          preserveSymlinks: true,
+        },
         configFile: false,
         ...userConfig,
       });

Reply:
If we put `resolve.preserveSymlinks` in `tmpl/vite.renderer.config.ts`, users can remove it at any time.

```diff
import { type UserConfig, defineConfig } from 'vite';
import { pluginExposeRenderer } from './vite.base.config';

// https://vitejs.dev/config
export default defineConfig((env) => {
  const { root, mode, forgeConfigSelf } = env as ForgeConfigEnv<'renderer'>;
  const name = forgeConfigSelf.name ?? '';

  return {
    root,
    mode,
    base: './',
    build: {
      outDir: `.vite/renderer/${name}`,
    },
    plugins: [pluginExposeRenderer(name)],
+   resolve: {
+     preserveSymlinks: true,
+   },
    clearScreen: false,
  } as UserConfig;
});
```

Reply:
It seems to be caused by `fast-glob`? Maybe you can use `os.readdir(os.tmpdir());` instead?